devicesetup/src/main/java/io/particle/android/sdk/utils/ParticleSetupConstants.java

```diff
@@ -0,0 +1,7 @@
+package io.particle.android.sdk.utils;
+
+public class ParticleSetupConstants {
+
+    public static final String CUSTOMIZATION_TAG = "customization";
```

Comment:
Can this be string be moved somewhere to make it only package-visible?  Public Is Forever(TM), and I'd like to keep external reliance on this sort of thing to a minimum.